Frontend/library/package.json

```diff
@@ -13,28 +13,28 @@
         "build": "npm run build:cjs && npm run build:esm",
         "rebuild": "npm run clean && npm run build",
         "watch": "nodemon -V -d 3 --watch src --watch ../../Common/dist -e \"ts,js,mjs,cjs,json\" --exec \"npm run build:cjs\"",
-        "lint": "eslint src --ext .js,.jsx,.ts,.tsx",
+        "lint": "eslint src",
         "test": "jest --detectOpenHandles --coverage=true",
         "spellcheck": "cspell \"{README.md,.github/*.md,src/**/*.ts}\""
     },
     "devDependencies": {
         "@types/jest": "^29.5.14",
-        "@typescript-eslint/eslint-plugin": "^6.21.0",
-        "cspell": "^4.1.0",
-        "eslint": "^8.11.0",
-        "eslint-config-prettier": "^9.1.0",
-        "eslint-plugin-prettier": "^5.2.1",
+        "@types/webxr": "^0.5.21",
+        "cspell": "^8.17.3",
+        "eslint": "^9.20.0",
         "jest": "^29.7.0",
         "jest-environment-jsdom": "^29.7.0",
-        "prettier": "3.3.3",
+        "nodemon": "^3.1.9",
+        "rimraf": "^6.0.1",
         "ts-jest": "^29.2.5",
-        "typedoc": "^0.27.4",
-        "typescript": "^5.0.0"
+        "typescript": "^5.7.3",
+        "typescript-eslint": "^8.24.0"
     },
     "dependencies": {
-        "@epicgames-ps/lib-pixelstreamingcommon-ue5.5": "^0.1.7",
-        "@types/webxr": "^0.5.1",
-        "sdp": "^3.1.0"
+        "sdp": "^3.2.0"
+    },
+    "peerDependencies": {
+        "@epicgames-ps/lib-pixelstreamingcommon-ue5.5": "^0.1.7"
```

Comment:
This one is tricky. While having it as a peer dep is one option, that would mean dependent libs would need to specify common as a dep. I think in this case we want common as a real dep not a peer. So that way users of our lib don't need to bring in common themselves too.